addon/components/o-s-s/search-field.hbs

```diff
@@ -0,0 +1,10 @@
+<OSS::InputContainer @value={{this.value}} @placeholder={{this.placeholder}} @onChange={{this.onChange}}>
```

Comment:
You can also add a `...attributes` here. Can be useful to pass a `data-control-name` for example :)